[#21] [UI] As a user, I can see my survey questions

nkhanh44 opened the pull request:
#21 

## What happened

- Implement Survey Question Screen UI.
 
## Insight

- Dummy data will be removed in integrate PR.
 
## Proof Of Work
**iPhone 8:**
![Simulator Screen Shot - iPhone 8 - 2022-08-24 at 10 30 46](https://user-images.githubusercontent.com/25881847/186323103-6ece7eb2-d4e6-4e10-80d7-78a4105df26f.png)

- video:

https://user-images.githubusercontent.com/25881847/186351270-c76166ab-23c8-467f-b89c-cbad7aac3c87.mp4



**iPhone 12 ProMax:**
![Simulator Screen Shot - iPhone 12 Pro Max - 2022-08-24 at 10 30 07](https://user-images.githubusercontent.com/25881847/186323117-9fe65fa8-df4b-4bac-921e-1b3335f100f2.png)

- video:

https://user-images.githubusercontent.com/25881847/186351405-5a93cde7-82ce-4531-af0d-12c36d1badde.mp4

blyscuit commented on Survey/Sources/Presentation/Modules/SurveyQuestion/SurveyQuestionView.swift:
```diff
@@ -0,0 +1,115 @@
+        .padding(.top, 54.0)
+        .background(
+            // TODO: Remove dummy cover image url
+            Image(survey.coverImageURL)
```
Even known this is a dummy data, I think you can reuse `list.first?.coverImageURL` so that you don't have to include a new line of `let`.

blyscuit commented on Survey/Sources/Presentation/Modules/SurveyQuestion/SurveyQuestionView.swift:
```diff
@@ -0,0 +1,115 @@
+            isShowing: .constant(false),
+            text: .constant(""),
+            content: {
+                ZStack {
```
You won't need this ZStack for now.

blyscuit reviewed (commented):
Looks good. UI looks clean.

blyscuit reviewed (commented):
I suggest you attach a video to show the scrolling when working with scrollable UI next time 👍

nkhanh44 commented:
@blyscuit Thank you 😄. I fixed all at https://github.com/nkhanh44/ic-survey-ios/pull/51/commits/2fa00fd9fb7c763b20ce384718abc28e6c581bf7 and also added videos to show the scrolling in description.
